lx_anonymizer/names_adder.py

```
@@ -4,10 +4,10 @@
 from pathlib import Path
 import time
 import ast
-from device_reader import read_device, read_text_formatting
-from directory_setup import create_temp_directory
-from box_operations import make_box_from_device_list, make_box_from_name, extend_boxes_if_needed
-from custom_logger import get_logger
+from .device_reader import read_device, read_text_formatting
+from .directory_setup import create_temp_directory
+from .box_operations import make_box_from_device_list, make_box_from_name, extend_boxes_if_needed
```

Ruff (0.8.2)

`.box_operations.make_box_from_device_list` imported but unused

Remove unused import

(F401)